modules/installation-mirror-repository.adoc

```diff
@@ -174,6 +174,11 @@ $ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mir
 ----
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
++
+[NOTE]
+====
+Due to the inclusion of old images in some image indexes, running `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can pass in the `--continue-on-error` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed]. 
```

Comment:
In the other file you say "you can *use* the `--continue-on-error` option" which I think is more clear than "pass in"

Reply:
Fixed!

Reply:
Fixed. Added the content type.